src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/service/NotificationGenerator.java

```diff
@@ -2,9 +2,13 @@
 
 import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
 import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.callback.PostSubmitCallbackResponse;
 
 public interface NotificationGenerator {
 
     void generate(Callback<AsylumCase> callback);
 
+    default PostSubmitCallbackResponse getDefaultConfirmationMessage() {
```

Comment:
can we use new Java model class like for example Result or Message?
it would be good to have model separation so Generator doesn't use typical controller object like PostSubmitCallbackResponse.
Could you change the naming to `getSuccessMessage`? we may need to add error message in future?

Reply:
I have added a new class and renamed the method